src/layouts/base.njk

```diff
@@ -32,16 +32,15 @@
     </nav>
   </header>
 
-    <main class="body-container">
-
+  <main>
+    <div class="body-container">
```

Comment:
I split this class name out to a separate `div`, but is that really the best thing? Is it better or worse if the class is on the `main` element?